### ISSUE TITLE
Support new instance type, m6i

### ISSUE THREAD

**Issue by francis-a** [boxfuse/boxfuse-issues#262]
AWS recently launched a new set of instance types, https://aws.amazon.com/blogs/aws/new-amazon-ec2-m6i-instances-powered-by-the-latest-generation-intel-xeon-scalable-processors/

Would it be possible to support these in boxfuse?

**Comment by axelfontaine** [comment]
Support for `m6i`, `c6i`, `r6i` and `db.t4g` instances is now available in the Console as well as the latest `1.34.0.1519` client.

**Comment by buma** [comment]
@axelfontaine 
I tried it with r6i and I got an error with `capacity=2-20:r6i.4xlarge:cpu30-60` set in boxfuse.conf. Previous instance type was r5.4xlarge and capacity was `capacity=2-20:r5.2xlarge:cpu30-60` and in both cases capacity is also set in `boxfuse run -capacity=${capacity_size}:${capacity_instance_type}:cpu30-70` seems that previous capacity in boxfuse.conf was out of sync with actually used one.

Where capacity_size is 2-10 and capacity_instance_type changed from r5.4xlarge to r6i.4xlarge.

I first thought it was an issue with old boxfuse: v.1.33.6.1510 but even after updating to v.1.35.0.1523 issue is the same.
```
ERROR: Invalid capacity info: 2-10:r6i.4xlarge:cpu30-60 => change it to one of the following formats:
  5:t2.small                       => ensure there are always 5 t2.small instances running
  2-10:c4.large:cpu30-70           => auto-scale between 2 (min) and 10 (max) c4.large instances
                                        based upon average CPU load over the last 300 seconds
                                        scale in at 30% and below, scale out at 70% and above
  2-10:c4.large:cpu30-70:60        => auto-scale between 2 (min) and 10 (max) c4.large instances
                                        based upon average CPU load over the last 60 seconds
                                        scale in at 30% and below, scale out at 70% and above
  2-10:c4.large:netin1024-8300     => auto-scale between 2 (min) and 10 (max) c4.large instances
                                        based upon incoming network traffic per instance over the last 300 seconds
                                        scale in at 1024 KB and below, scale out at 8300 KB and above
  2-10:c4.large:netin1024-8300:60  => auto-scale between 2 (min) and 10 (max) c4.large instances
                                         based upon incoming network traffic per instance over the last 60 seconds
                                         scale in at 1024 KB and below, scale out at 8300 KB and above
  2-10:c4.large:netout1024-8300    => auto-scale between 2 (min) and 10 (max) c4.large instances
                                        based upon outgoing network traffic per instance over the last 300 seconds
                                        scale in at 1024 KB and below, scale out at 8300 KB and above
  2-10:c4.large:netout1024-8300:60 => auto-scale between 2 (min) and 10 (max) c4.large instances
                                        based upon outgoing network traffic per instance over the last 60 seconds
                                        scale in at 1024 KB and below, scale out at 8300 KB and above
```

**Comment by axelfontaine** [comment]
@buma Ensure you are on the latest client (`1.35.0.1523`) and it will then work as expected.

**Comment by buma** [comment]
@axelfontaine It looks to me that latest client is used. I upgraded boxfuse the first time I got an error and version reported when image is built is latest one: 
![image](https://user-images.githubusercontent.com/1055967/144821518-4ed853bf-944c-4303-a3c9-074d3cf668b5.png)

**Comment by axelfontaine** [comment]
Arrrgh! You're right! It looks like we failed to integrate our new capacity parsing code adding support for CPU-type suffixes in that release 🤦

Fixed in `1.35.1.1524`.

**Comment by buma** [comment]
@axelfontaine Thank you for the quick change. It works great now.
